lib/src/countly_flutter.dart

```diff
@@ -534,6 +548,7 @@ class Countly {
 
   /// Update session for manual session handling.
   /// This method needs to be called for updating a session only if manual session handling is enabled by calling the 'enableManualSessionHandling' method of 'CountlyConfig'.
+  /// returns the error or success message
 
```

Comment:
shouldnt those new lines would be removed?

Reply:
The pull request is for adding comments to functions that don't return void to explain what they return.

@ArtursKadikis should I remove the new lines?

Reply:
yes. on it.